fdbclient/StorageServerInterface.h

```diff
@@ -441,7 +441,11 @@ struct SplitMetricsRequest {
 // Should always be used inside a `Standalone`.
 struct ReadHotRangeWithMetrics {
 	KeyRangeRef keys;
+	// density refers to the ratio of bytes sent(because of the read) and bytes on disk.
+	// For example if key range [A, C) has byte size 100 bytes on disk and was read 100 times,
```

Comment:
better example:
key range [A, B) and [B, C) respectively has byte size 100 bytes on disk.
Key range [A,B) was read 30 times.
The density for key range [A,C) is 30 * 100 / 200 = 15

Reply:
Done.